java/src/jmri/jmrit/display/PositionablePropertiesUtil.java

```diff
@@ -113,29 +116,32 @@ public void actionPerformed(ActionEvent e) {
     //JLabel example;
 
     ImageIcon[] images;
-    String[] _fontcolors = {"Black", "Dark Gray", "Gray", "Light Gray", "White", "Red", "Orange", "Yellow", "Green", "Blue", "Magenta"};
+    String[] _fontcolors = {"Black", "Dark Gray", "Gray", "Light Gray", "White", "Red", "Orange", "Yellow", "Green", "Blue", "Magenta"}; // NOI18N
     String[] _backgroundcolors;
     JComboBox<Integer> fontColor;
     JComboBox<Integer> backgroundColor;
     JTextField fontSizeField;
 
-    String[] _justification = {"Left", "Right", "Center"};
+    String[] _justification = {"Left", "Right", "Center"}; // TODO I18N
     JComboBox<String> _justificationCombo;
 
+    /**
+     * Create and fill in the Font (Decoration) tab of the UI.
+     */
     void textPanel() {
         _textPanel = new JPanel();
         _textPanel.setLayout(new BoxLayout(_textPanel, BoxLayout.Y_AXIS));
         JPanel fontColorPanel = new JPanel();
-        fontColorPanel.add(new JLabel("Font Color"));
+        fontColorPanel.add(new JLabel(Bundle.getMessage("FontColor") + ": "));
 
         JPanel backgroundColorPanel = new JPanel();
-        backgroundColorPanel.add(new JLabel("Background Color"));
+        backgroundColorPanel.add(new JLabel(Bundle.getMessage("FontBackgroundColor") + ": "));
         Color defaultLabelBackground = backgroundColorPanel.getBackground();
         _backgroundcolors = new String[_fontcolors.length + 1];
         for (int i = 0; i < _fontcolors.length; i++) {
             _backgroundcolors[i] = _fontcolors[i];
         }
-        _backgroundcolors[_backgroundcolors.length - 1] = "None";
+        _backgroundcolors[_backgroundcolors.length - 1] = Bundle.getMessage("ColorClear"); // I18N using Bundle.getMessage("ColorClear");
```

Comment:
No, it's stores as ``hasBackground="yes" redBack="255" greenBack="255" blueBack="255"`` so no problem.
However, in the present code when you choose Clear as the bg color in the popup menu of a Sensor Icon (i.e. to remove a background color), instead it saves as the color of the background (works as expected storing Clear/None with a "Sensor text icon").